components/Banner/Banner.jsx

```diff
@@ -0,0 +1,18 @@
+import stylesbanner from '@/styles/banner.module.css'
```

Comment:
Please use the patter `styles` and the initial capital letter for the css file

Reply:
Done!!